src/components/GrampsjsPlaceRefs.js

```diff
@@ -111,6 +127,20 @@ export class GrampsjsPlaceRefs extends GrampsjsEditableTable {
     }
   }
 
+  _handleEditClick(handle) {
+    const obj = this.data.find(p => p.ref === handle)
+    this.dialogContent = html`
+      <grampsjs-form-placeref
+        @object:save="${e => this._handlePlaceRefUpdate(e, obj)}"
+        @object:cancel="${this._handlePlaceRefCancel}"
+        .appState="${this.appState}"
+        .data="${obj}"
```

Comment:
Don't you also have to set the `.place` property for the selected place to be displayed correctly?